edxval/tests/test_views.py

```diff
@@ -498,7 +498,7 @@ def test_post_videos(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(
             response.data.get("edx_video_id")[0],
-            "This field must be unique."
+            u'video with this edx video id already exists.'
```

Comment:
The code:
https://github.com/encode/django-rest-framework/blob/3.6.3/rest_framework/exceptions.py#L121-L140